util/dvsim/Deploy.py

```diff
@@ -171,6 +171,37 @@ def construct_cmd(self):
         # self.cmd += " > " + self.log + " 2>&1 &"
         return cmd
 
+    def is_equal(self, item):
+        '''Returns True if `item` is equivalent to `self`.
+
+        Checks if item is identical to self in all aspects except for the name,
+        which would obviously be different. We only need to check the `cmd`
+        parameter & the exports.
+        '''
```

Comment:
I think this needs a more specific name: this isn't checking for equality of `Deploy` objects (if it were, I'd be suggesting you overload `__eq__`), but instead checking whether the `Deploy` objects describe the same task. Maybe `is_equivalent_job` or something?

Reply:
Done.

Reply:
This doesn't gate the review, but this test is "I have a projection from Deploy objects to commands+exports. Do these two Deploy objects project to the same thing?" Your new name is `is_equivalent_to`. It's probably better than `is_equal`, because it doesn't talk about equality, but it still doesn't give any notion of what the equivalence might be.

Reply:
Is the description not clear enough?

Reply:
Well, I think the names matter. But I'm not turning this into a huge argument: if you want to use `is_equivalent_to`, let's go with that.

Reply:
Sorry, I am drawing a blank on how to best name this method. I am happy to change this to whatever you prefer.

Reply:
Well, I suggested `is_equivalent_job` above. I'm also struggling to name it, but I guess that's probably a sign that I'm not quite sure what it does.

Reply:
Okay, done.